content/developers/dogstatsd/data_types.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Data Types and Tags
+kind: documentation
+description: Overview of the features of DogStatsD, including data types and tagging.
+further_reading:
+- link: "developers/dogstatsd"
+  tag: "Documentation"
+  text: Introduction to DogStatsD
+- link: "developers/libraries"
+  tag: "Documentation"
+  text: Official and Community-contributed API and DogStatsD client libraries
+- link: "https://github.com/DataDog/dd-agent/blob/master/dogstatsd.py"
+  tag: "Github"
+  text: DogStatsD source code
+---
+
+While StatsD accepts only metrics, DogStatsD accepts all three of the major Datadog data types: metrics, events, and service checks. This section shows typical use cases for each type, and introduces tagging, which is specific to DogStatsD.
+
+Each example is in Python using the [official Datadog Python client][2], but each data type shown is supported similarly in [other DogStatsD client libraries][1].
+
+## Metrics
+
+Counters, gauges, and sets are familiar to StatsD users. Histograms are specific to DogStatsD. Timers, which exist in StatsD, are a sub-set of histograms in DogStatsD.
```

Comment:
Should we capitalise metrics types? i.e `Gauge`, `Sets` ... ?

Reply:
Imho this question is out of scope for this PR. Leaving them lower-case for now.